src/kt_class.cpp

```diff
@@ -8,6 +8,7 @@ KtClass::KtClass(jni::JObject p_wrapped, jni::JObject& p_class_loader) :
     jni::Env env { jni::Jvm::current_env() };
     name = get_name(env);
     super_class = get_super_class(env);
+    base_godot_class = get_base_godot_class(env);
```

Comment:
I'm just thinking: couldn't we combine those calls into one? because as of now we make three subsequent calls from cpp to jvm for each class right?

Reply:
This happens only once per class during registration, probably not worth optimizing.

Reply:
This can be optimized later if we see we have long engine boot times.

Reply:
ok for me